src/components/color_generator/index.js

```diff
@@ -60,101 +87,7 @@ class ColorGenerator extends Component {
           placeholder="Palette Name"
           maxLength="10"
         />
-        <fieldset className="color-field">
```

Comment:
HUGE delete, thank you for making my code dynamic. I had thought about it after the fact last night, but my primary goal was to get things usable and on the page.